src/cmd/maturity.go

```diff
@@ -30,7 +30,8 @@ There are multiple output formats that are useful
 		categoriesConn, err := client.ListCategories(nil)
 		cobra.CheckErr(err)
 		categories := categoriesConn.Nodes
-		data, err := client.ListServicesMaturity()
+		response, err := client.ListServicesMaturity(nil)
```

Comment:
@davidbloss the nilaway PR I have is not ready yet, but this is an example of a situation where we're vulnerable to nil pointer exceptions.

The `checkErr` should be done before accessing `response.Nodes` since `response` could be nil. 

Approving PR

Reply:
Shuffled ✅